### ISSUE TITLE
How to use IK controller in IsaaclabHandler?

### ISSUE THREAD

**Issue by 4evertutelary** [RoboVerseOrg/RoboVerse#201]
Dear contributor:
Is it possible to  use IK controller to step official openvla checkpoint preditcted action?

cmd:
python roboverse_learn/eval.py --sim=isaaclab --task=LiberoPickButter --algo openvla --checkpoint_path '/home/xxx/code/openvla1/openvla-7b-finetuned-libero-object'

main code change in eval.py:
            new_obs = {
                # "rgb": obs.cameras["camera0"].rgb,
                "head_cam": obs.cameras["camera0"].rgb,
                # "joint_qpos": obs.robots[args.robot].joint_pos,
            }

            # images_list.append(np.array(new_obs["rgb"].cpu()))
            images_list.append(np.array(new_obs["head_cam"].cpu()))
            # action = policyRunner.get_action(new_obs)
            action = policyRunner.predict_action(new_obs)

action should be: [Δx, Δθ, ΔGrip]
![Image](https://github.com/user-attachments/assets/fe008c79-6552-40fe-93f4-c61c62f86480)

**Comment by 4evertutelary** [comment]
Seems IsaaclabHandler only support joint controller at present:

    ############################################################
    ## Gymnasium main methods
    ############################################################
    def step(self, action: list[Action]) -> tuple[Obs, Reward, Success, TimeOut, Extra]:
        self._actions_cache = action
        actuator_names = [k for k, v in self.robot.actuators.items() if v.actionable]
        action_env_tensors = torch.zeros((self.num_envs, len(actuator_names)), device=self.env.device)
        for env_id in range(self.num_envs):
            action_env = action[env_id]
            for i, actuator_name in enumerate(actuator_names):
                action_env_tensors[env_id, i] = torch.tensor(
                    action_env["dof_pos_target"][actuator_name], device=self.env.device
                )

        _, _, _, time_out, extras = self.env.step(action_env_tensors)
